UpdateTagJenkins.py

```diff
@@ -0,0 +1,62 @@
+import sys
```

Comment:
We have a new script for this, also it contains some changes that are not here. Maybe we should use that version?